en/docs/chapter_sorting/bucket_sort.md

```diff
@@ -1,12 +1,12 @@
 # Bucket sort
 
-The previously mentioned sorting algorithms are all "comparison-based sorting algorithms," which sort by comparing the size of elements. Such sorting algorithms cannot surpass a time complexity of $O(n \log n)$. Next, we will discuss several "non-comparison sorting algorithms" that can achieve linear time complexity.
+The previously mentioned sorting algorithms are all "comparison-based sorting algorithms," which sort elements by comparing their values. Such sorting algorithms cannot perform better than time complexity of $O(n \log n)$. Next, we will discuss several "non-comparison sorting algorithms" that could achieve linear time complexity.
```

Comment:
"cannot perform better than time complexity of" -> "cannot have better time complexity of"